iidm/iidm-impl/src/test/java/eu/itesla_project/iidm/network/impl/HvdcLineImplTest.java

```diff
@@ -0,0 +1,31 @@
+package eu.itesla_project.iidm.network.impl;
+
+import eu.itesla_project.iidm.network.HvdcLine;
+import eu.itesla_project.iidm.network.Network;
+import eu.itesla_project.iidm.network.test.HvdcTestNetwork;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class HvdcLineImplTest {
```

Comment:
Rename class to HvdcLineTest (impl suffix is given by the module name) and is redundant here